Oleksii_Fedorchuk/HW12_Oleksii_Fedorchuk/task1.py

```diff
@@ -0,0 +1,52 @@
+class Company:
+    """Class of the company description"""
+
+    def __init__(self, company_name: str, company_website: str, company_description: str, company_products: str):
+        """Initiation"""
+
+        self.__company_name = company_name
+        self.__company_website = company_website
+        self.__company_description = company_description
+        self.__company_products = company_products
+
+    def company_name(self):
+        """Show company name"""
+
+        company_name = self.__company_name
+        print(company_name)
```

Comment:
please use @property instead. It is related to all similar methods in this class

Also implement setters all private fields